db/migrate/20160406142950_fix_versions_item_type_for_steps.rb

```diff
@@ -0,0 +1,9 @@
+class FixVersionsItemTypeForSteps < ActiveRecord::Migration
+  def up
+    execute "UPDATE versions SET item_type = 'Steps::Approval' WHERE item_type = 'Approval'"
+  end
+
+  def down
+    execute "UPDATE versions SET item_type = 'Approval' WHERE item_type = 'Steps::Approval'"
```

Comment:
This rollback affects _all_ `Steps::Approval` rows, not just those updated in the migration. Maybe limit with a date clause? Or am I being overly cautious? (Probably.)